tests/e2e/mod.rs

```diff
@@ -1,20 +1,22 @@
-use std::env;
-use std::path::Path;
-use std::process::Command;
-use std::time::Duration;
-use std::{collections::HashMap, path::PathBuf};
-
 use anyhow::Error;
 use camino::Utf8PathBuf;
 use dircpy::CopyBuilder;
 use duct::cmd;
 use git2::Repository;
 use portpicker::pick_unused_port;
+use regex::Regex;
 use reqwest::Client;
 use rstest::*;
 use serde::Deserialize;
 use serde_json::json;
+use std::env;
+use std::io::{BufRead, BufReader};
+use std::path::Path;
+use std::process::ChildStderr;
+use std::time::Duration;
+use std::{collections::HashMap, path::PathBuf};
```

Comment:
NIT: these should be separate at the top.

Reply:
Cool, I've used some `rustfmt` config from `router` to split them out, let me know if that's what you wanted